### ISSUE TITLE
A few current questions about this package

### ISSUE THREAD

**Issue by so1ve** [halvardssm/oak-middleware-jwt#8]
Hello Halvard,
I found that the current `oak-middleware-jwt` has the following problems:
1. The version of `djwt` used is too low, you should upgrade to v2.4 to solve this problem when using it under Deno v1.18.1 https://github.com/timonson/djwt/issues/64
````
error: TS2769 [ERROR]: No overload matches this call.
   The last overload gave the following error.
     Argument of type '"jwk"' is not assignable to parameter of type '"raw" | "pkcs8" | "spki"'.
     "jwk",
     ~~~~~~
     at https://deno.land/x/god_crypto@v1.4.9/src/rsa/rsa_wc.ts:55:5

TS2771 [ERROR]: The last overload is declared here.
         at asset:///lib.dom.d.ts:13442:5
````
The new version of `djwt` does not use the `god_crypto` library and removes some exports used in oak-middleware-jwt.
2. The new version of Oak is not supported. `Middleware` types and `RouterMiddleware` types in the new version cannot be converted to and from the types in the v8.0.0 version of Oak.

I [forked this repo](https://github.com/so1ve/oak-middleware-jwt) and modified it a bit to make it work under v10.2.0 of Oak and v1.18.1 of Deno, but it's not a perfect solution.

**Comment by halvardssm** [comment]
Hi Ray! Ah, yeah, this repo hasn't received much love lately... Would you like to submit your branch for a PR (don't think I will merge it, but it's good to have the overview), and then I can get to work with updating this repo?

**Comment by halvardssm** [comment]
Small update Ray! Here is a PR I smashed together. Let me know if this solves all of your issues! https://github.com/halvardssm/oak-middleware-jwt/pull/9

**Comment by halvardssm** [comment]
New release has been published (2.2.0), so I will close this issue. Let me know if you still have any issues which were not solved so that we can create a new issue 💪🏻

**Comment by so1ve** [comment]
Sorry, I didn't see the reply here for the first time... I will try this new version
